src/main/java/io/pinecone/commons/IndexInterface.java

```diff
@@ -25,7 +25,7 @@
  * @param <X> The return type for delete operations.
  * @param <Y> The return type for describing index stats.
  */
```

Comment:
Could you add the `Z` generic parameter to the docstring above?
```suggestion
 * @param <X> The return type for delete operations.
 * @param <Y> The return type for describing index stats.
 * @param <Z> The return type for the list operation.
 */
```

Reply:
done!